WinUIGallery/Navigation/NavigationRootPage.xaml.cs

```diff
@@ -226,7 +227,7 @@ private void AddNavigationMenuItems()
                 NavigationViewControl.MenuItems.Add(itemGroup);
             }
 
-            NewControlsItem.Loaded += OnNewControlsMenuItemLoaded;
+            Home.Loaded += OnNewControlsMenuItemLoaded;

```

Comment:
Should this be called OnHomeMenuItemLoaded now?

Reply:
Good catch. Turns out there's a ton of NewControls* names that will need to be updated as well. I will do that in a separate PR.